### ISSUE TITLE
Is it possible to use more than 8 colors?  

### ISSUE THREAD

**Issue by alexei-28** [wolray/symbol-overlay#100]
Here customize:

![cusomize](https://github.com/wolray/symbol-overlay/assets/303539/a5544ebb-3301-43ce-a29e-31a858524796)

And here example:
![example](https://github.com/wolray/symbol-overlay/assets/303539/46c7b022-58b5-41e0-9d96-81aadf045c41)

Is it possible to add more colors? At least 10 colors.

**Comment by purcell** [comment]
Yes, I think you can define some extra faces and then add them to `symbol-overlay-faces`.

**Comment by alexei-28** [comment]
> Yes, I think you can define some extra faces and then add them to `symbol-overlay-faces`.

I try like this, but it not help. Only 8 colors :-(

![image](https://github.com/wolray/symbol-overlay/assets/303539/fc692b22-4abd-4659-8cfe-058c213f8db4)

**Comment by purcell** [comment]
The buffer on the left doesn't show that the changes have been saved. Can you confirm that the variable `system-overlay-faces` now actually contains `symbol-overlay-face-9` and `symbol-overlay-face-10`? (Use `describe-variable`, ie. `C-h v`)

**Comment by alexei-28** [comment]
> The buffer on the left doesn't show that the changes have been saved. Can you confirm that the variable `system-overlay-faces` now actually contains `symbol-overlay-face-9` and `symbol-overlay-face-10`? (Use `describe-variable`, ie. `C-h v`)

When (in the left buffer) I select State -> Save for Future Sessions" I get error:

`custom-variable-mark-to-save: Saving symbol-overlay-faces: Invalid face: symbol-overlay-face-9
`

**Comment by purcell** [comment]
You would also need to define the face using `defface`. Add something like this to your startup files:
```el
(with-eval-after-load 'symbol-overlay
  (defface symbol-overlay-face-9
    '((t (:background "turquoise" :foreground "black")))
    "Symbol Overlay default candidate 8"
    :group 'symbol-overlay)

  (defface symbol-overlay-face-10
    '((t (:background "turquoise" :foreground "black")))
    "Symbol Overlay default candidate 8"
    :group 'symbol-overlay)

  (add-to-list 'symbol-overlay-faces 'symbol-overlay-face-9 t)
  (add-to-list 'symbol-overlay-faces 'symbol-overlay-face-10 t))
```

then all you need to do in the `Custom` interface is customize those two faces.

**Comment by purcell** [comment]
What was happening for you above was this:
- Only faces defined using `defface` are accepted when `symbol-overlay-faces` is changed using `Customize`
- Customized face definitions are accepted but ignored if the faces have not been defined using `defface`.
